core/services/fluxmonitor/flux_monitor_internal_test.go

```diff
@@ -0,0 +1,29 @@
+package fluxmonitor
+
+import (
+	"github.com/ethereum/go-ethereum/common"
+	"github.com/smartcontractkit/chainlink/core/services/eth"
+)
+
+func (fm *concreteFluxMonitor) MockLogBroadcaster() *mockLogBroadcaster {
```

Comment:
Can this not be done with `stretchr/mocks` ?

Reply:
Maybe creating the mock object can be done with that, but as far as I know adding a method to the type internal to the package cannot(still haven't used strechr that much).

I'd like to circle back and fix all of this up more, but this small change fixes a regression and is blocking someone right now.